pixels-common/src/main/java/io/pixelsdb/pixels/common/physical/StreamPath.java

```diff
@@ -0,0 +1,37 @@
+package io.pixelsdb.pixels.common.physical;
+
+import static java.util.Objects.requireNonNull;
+
+public class StreamPath
+{
+    private String host;
+    private int port;
+    public boolean valid = false;
+
+    public StreamPath(String path)
+    {
+        requireNonNull(path);
+        if (path.contains(":///"))
```

Comment:
why ':///' instread of '://'?

Reply:
refer to FilePath.java
```java
public FilePath(String path)
    {
        requireNonNull(path);
        if (path.startsWith("file:///"))
        {
            valid = true;
            realPath = path.substring(path.indexOf("://") + 3);
        }
        else if (path.startsWith("/"))
        {
            valid = true;
            realPath = path;
        }

        if (valid)
        {
            File file = new File(realPath);
            isDir = file.isDirectory();
        }
    }
```

Reply:
Local file paths always start with '/,' hence there are three /// in total.
However, streaming paths should start with the destination hostname.

Reply:
Local file paths always start with '/,' hence there are three /// in total.
However, streaming paths should start with the destination hostname.